packages/account/src/Components/poi/idv-document-submit/idv-document-submit.tsx

```diff
@@ -107,10 +107,23 @@ const IdvDocumentSubmit = ({
 
         const data = await WS.setSettings(request);
 
-        const generic_error_message = localize('Sorry, an internal error occurred. Click Verify to try again.');
+        const generic_error_message = (
+            <Localize i18n_default_text='Sorry, an internal error occurred. Click Verify to try again.' />
```

Comment:
```suggestion
<Localize i18n_default_text='Sorry, an internal error occurred. Click verify to try again.' />
```

Reply:
capital letter here is according to design